### DISCUSSION TITLE
Someip SD Callback handling

### DISCUSSION THREAD

**Discussion by omar987** [COVESA/vsomeip#475] [Q&A]
Hello Community, 

I want to ask about the possibility to register on SOME/IP service discovery packet (Find, Subscribe/Subscribe_ack) and get them into a callback in my application. 

Thanks in advance.

**Comment by NagarjunaReddyG5262** [comment]
Hi I also have this very similar question. 
If there are two vsomeip instances running. One application acts as server and one acts as client. 
Client application calls subscribe method to subscribe to a event group. 
How the server will know that client wants to subscribe to that event group? I didn't find any call back for this as we have for Request and Response. 

If any one have idea on this and understand my scenario, any help would be appreciated. 
Thanks
